index.html

```diff
@@ -194,35 +252,35 @@ <h1>
                 <textarea id="msg" name="user_message" placeholder="Write me something..." maxlength="500" required></textarea>
                 <button class="submitform" type="submit">Get in touch</button>
```

Comment:
- [OPTIONAL] You did a great job implementing the contact form and all the changes requested by the previous reviewer. 👍 👍 , I noticed that you did not add interactive states effects to your submit button, like the `Hover` and `Enabled`, Please consider adding this as it greatly improves the user experience of your website. 😄 😸